datadog_cluster_agent/assets/configuration/spec.yaml

```
@@ -0,0 +1,12 @@
+name: Datadog Cluster Agent
+files:
+- name: datadog_cluster_agent.yaml
+  options:
+  - template: init_config
+    options:
+      - template: init_config/openmetrics_legacy
+  - template: instances
+    options:
+      - template: instances/openmetrics_legacy
+        overrides:
+          prometheus_url.value.example: http://localhost:5000/metrics
```

```suggestion
          prometheus_url.value.example: http://localhost:5000/metrics

- name: auto_conf.yaml
  options:
  - template: ad_identifiers
    overrides:
      value.example:
        - cluster-agent
  - template: init_config
    options: []
  - template: instances
    options:
    - name: prometheus_url
      description: The URL where your application metrics are exposed by Prometheus.
      value:
        example: http://%%host%%:5000/metrics
        type: string
```

You can include auto_conf.yaml here as well

---

Thanks. I added it and regenerated the sample file.